pkg/ivd/ivd_protected_entity_type_manager.go

```diff
@@ -378,11 +380,32 @@ func (this *IVDProtectedEntityTypeManager) ReloadConfig(ctx context.Context, par
 		this.cnsManager.ResetManager(reloadedVc, cnsClient)
 	}
 
-	err = disklib.Init(vsphereMajor, vSphereMinor, disklibLib64)
-	if err != nil {
-		return errors.Wrap(err, "Could not initialize VDDK during config reload.")
+	// check whether customized vddk config is provided.
+	// currently only support user to modify vixdisklib nfc log level and transport log level
+	if _, ok := params[vddkconfig]; !ok {
+		this.logger.Info("No customized vddk log level provided, set vddk log level as default.")
```

Comment:
Please double check all logging statement in your PR and remove the end period in each of them.

>this.logger.Info("No customized vddk log level provided, set vddk log level as default")